ingest_TSV.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>TSV Ingest</title>
+    <script src="https://d3js.org/d3.v5.js"></script>
+    <script src="https://d3js.org/d3-fetch.v1.min.js"></script>
+    <script src="https://apis.google.com/js/api.js"></script>
+    <script src="Api.js"></script>

```

Comment:
Same comment as with your other html file, keep the skeleton that was created for batch_ingest.html in the core repo, and only include actual files, and make sure to include the css file.